routes/web.php

```diff
@@ -36,4 +36,6 @@
     Route::get('/countries/{id}/delete', 'CountryController@destroy')->name('adminCountriesDelete');
 
     Route::get('/cities', 'CityController@index')->name('adminCities');
+    Route::get('/cities/create', 'CityController@create')->name('adminCitiesCreate');
+    Route::post('/cities', 'CityController@store')->name('adminCities');
```

Comment:
Name route change to `adminCitiesStore`. adminCities is duplicate with list route